website/app/styles/pages/application/footer.scss

```diff
@@ -7,7 +7,7 @@
   // z-index: var(--doc-z-index-footer);
   grid-area: footer;
 
-  body.application:not(.index) & {
+  body.application:not(.index, .error, .patterns) & {
```

Comment:
Question: I wonder if it would make sense to instead add a "show-sidebar" or else "hide-sidebar" type class to certain pages?

Reply:
Not sure if that would simplify anything though...